pom.xml

```diff
@@ -3,9 +3,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.amazonaws.services.glue.dataquality</groupId>
+    <groupId>software.amazon.glue</groupId>
     <artifactId>dqdl</artifactId>
-    <version>1.0.0</version>
+    <version>0.9.0</version>
```

Comment:
any reason for this change?

Reply:
We don't have the latest changes in the package, so I cannot deem it v1. Once we merge all the latest changes in, we can release a new version - v1.0.0